scripts/updateGeneratedFiles.ts

```diff
@@ -3,19 +3,26 @@ import fs from "fs/promises";
 import path from "path";
```

Comment:
If you wanted you could put the scripts into a `packages` folder and also have them part of the yarn workspace. Could even expose them with `bin` fields in package.json

Reply:
ok will have a look at that in another PR, this one is big enough